instructions/kiali/kiali.md

```diff
@@ -49,8 +51,8 @@ You should now see the application graph showing traffic being created by our sc
 
 Within the Kiali UI select the *Distributed Tracing* option from the left hand navigation and then choose
 
-* Namespace: tutorial
-* Service: customer.tutorial
```

Comment:
Down below, "please keep you Kiali open" should be please keep your

Reply:
Kiali provides additional capabilties

capabilities

Reply:
@brian-avery Submit a new PR for these changes, this one is only about the namespace change.